beets/autotag/mb.py

```diff
@@ -270,8 +270,8 @@ def track_info(recording, index=None, medium=None, medium_index=None,
         info.arranger = u', '.join(arranger)
 
     # Supplementary fields provided by plugins
-    extra_trackdatas = plugins.send('mb_track_extract', data=recording)
-    for extra_trackdata in extra_trackdatas:
+    extra_trackdatasets = plugins.send('mb_track_extract', info=recording)
```

Comment:
Isn't the parameter called data in the docs?

https://github.com/beetbox/beets/blame/master/docs/dev/plugins.rst#L255

Reply:
yes, sorry I screwed up

Reply:
it should be fine now